### PR TITLE
migration to androidX project #147

### PR DESCRIPTION
**Pull request by AndroFlo**
(make with the android studio refactor tool)

### CODE REVIEW

File: tests/build.gradle [hunk 1]
```diff
@@ -4,21 +4,21 @@ apply plugin: 'realm-android'
 
 android {
     compileSdkVersion sdkVersion
-    buildToolsVersion buildTools
+    buildToolsVersion '28.0.2'
```
Comment (cmelchior) [inline tests/build.gradle]: Please update the variable value instead of replacing it with a hardcoded value.

Comment (AndroFlo) [reply tests/build.gradle]: Hi @cmelchior i replaced buildToolsVersion  hardCoded in my last commit

**Comment by Yayo-Arellano** [comment]
Good day

I updated to androidX and realize my project will not compile anymore. Is it possible to add this PR to the SNAPSHOTS so we can start using/testing??

**Comment by cmelchior** [comment]
Replaced by #154

**Pull request closed without merging**